src/include/cell_inline.h

```diff
@@ -1235,6 +1235,11 @@ __cell_data_ref(WT_SESSION_IMPL *session, WT_PAGE *page, int page_type,
             return (0);
         break;
     case WT_CELL_VALUE_OVFL:
+        /*
+         * Encourage checkpoint to race with reading the onpage value. If we have an overflow item,
+         * it may be removed by checkpoint concurrently.
+         */
+        __wt_timing_stress(session, WT_TIMING_STRESS_SLEEP_BEFORE_READ_ONPAGE, NULL);
```

Comment:
As this timing stress is specific to the overflow, How about renaming the option to contain the overflow also?